packages/plugins/RSS3/src/SNSAdaptor/components/FeedCard/LiquidityCard.tsx

```diff
@@ -97,18 +110,30 @@ export const LiquidityCard: FC<TokenFeedCardProps> = ({ feed, ...rest }) => {
                     }}
                 />
             </Typography>
-            {targetToken ? (
-                <div className={cx(classes.token, verbose ? classes.verboseToken : null)}>
-                    <Image classes={{ container: classes.tokenIcon }} src={targetToken.image} height={40} width={40} />
-                    <Typography className={cx(classes.value, isSupply ? classes.supply : classes.withdraw)}>
-                        {isSupply ? '+ ' : '- '}
-                        {t.token_value({
-                            value: formatValue(targetToken),
-                            symbol: targetToken.symbol,
-                        })}
-                    </Typography>
+            <div className={classes.horizonCenter}>
```

Comment:
Do you need to hide this chunk if `metadata?.tokens.length` is `undefined`?

Reply:
OK.